src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -147,23 +149,30 @@ class LocalBackend extends Backend with SharedFileSystem with LazyLogging {
 
   private def runSubprocess(backendCall: BackendCall): ExecutionResult = {
     val tag = makeTag(backendCall)
-    val (_, stdoutWriter) = backendCall.stdout.fileAndWriter
-    val (_, stderrWriter) = backendCall.stderr.fileAndWriter
+    val stdoutTailed = backendCall.stdout.tailed(100)
+    val stderrTailed = backendCall.stderr.tailed(100)
```

Comment:
The tail of the stderr will be used in the future to aid debugging docker. After merge conflicts cool down, I'll re-add the tail of  the logs to the `Failure` case "below", when we don't get an "rc" file at all. Hopefully this will allow us to debug what's going on when docker fails to launch.

I removed that functionality for now based on PR comments. Here is the reverted line:
https://github.com/broadinstitute/cromwell/pull/200#discussion-diff-40596766R199